examples/layout_headerbar_example.py

```diff
@@ -19,20 +19,19 @@ def __init__(self):
         image = Gtk.Image.new_from_gicon(icon, Gtk.IconSize.BUTTON)
         button.add(image)
         hb.pack_end(button)
-        
+
         box = Gtk.Box(orientation=Gtk.Orientation.HORIZONTAL)
         Gtk.StyleContext.add_class(box.get_style_context(), "linked")
 
-        button = Gtk.Button()
-        button.add(Gtk.Arrow(Gtk.ArrowType.LEFT, Gtk.ShadowType.NONE))
+        button = Gtk.Button.new_from_icon_name("pan-start-symbolic", Gtk.IconSize.MENU)
```

Comment:
"pan-start-symbolic" is a different symbol from Gtk.ArrowType.LEFT

Reply:
I know but this was the suggested replacement in my searching